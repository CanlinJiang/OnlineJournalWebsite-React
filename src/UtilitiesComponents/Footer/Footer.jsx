import React from 'react'
import {Link} from "react-router-dom"
import styles from "./Footer.module.css"

export default function Footer() {
    


    return (
        <nav>
            <ul className={styles.navigation}>
                <li id = "aboutMe" ><Link to="/aboutme">About me</Link></li>
                <li id = "contactMe"><Link to="/contactme">Contact me</Link></li>
            </ul>    
        </nav>
    )
}
