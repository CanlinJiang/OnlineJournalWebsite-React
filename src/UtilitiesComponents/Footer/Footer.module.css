.navigation {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center;
    list-style-type: none;
    font-size: 0.9rem;
    padding: 0;
    margin: 0;
}

.navigation li{
    padding:2px 2px;
    border-right: 2px solid rgb(59, 56, 56);
}

.navigation li:last-child{
    padding:2px 2px;
    border-right: none;
}


.navigation a{
    display: block;
    color: black;
    text-align: center;
    text-decoration: none;
    padding: 0px 10px;
}

.navigation a:hover{
    display: block;
    color: black;
    text-align: center;
    text-decoration: underline;
    padding: 0px 10px;
}


