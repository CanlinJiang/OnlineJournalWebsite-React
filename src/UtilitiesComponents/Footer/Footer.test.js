import Footer from "./Footer"
import React from "react";

describe("Footer tests", () => {
    it("renders correctly", () => {
        
      const { container } = render(<Footer />);
      expect(container).toMatchSnapshot();
    });
  
  });