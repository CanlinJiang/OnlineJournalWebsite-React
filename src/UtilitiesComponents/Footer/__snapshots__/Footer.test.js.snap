// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests renders correctly 1`] = `
<div>
  <nav>
    <ul
      class="navigation"
    >
      <li
        id="aboutMe"
      >
        <a
          href="/aboutme"
        >
          About me
        </a>
      </li>
      <li
        id="contactMe"
      >
        <a
          href="/contactme"
        >
          Contact me
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
